webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+    entry: './src/index.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, 'dist'),
+    },
+    plugins: [
+        new CopyPlugin({
+            patterns: [
+                { from: 'src/public/index.html', to: 'index.html' },
+                { from: 'src/public/style.css', to: 'style.css' },
+            ],
+        }),
+    ],
+    module: {
+        rules: [
+            {
+                test: /\.m?js$/,
```

Comment:
This does the transpilation of the es2020 code (the latest version of javascript) to a version that is understood by the browser (an older version).